### PR TITLE
Safari Extension Conversion

### PR DESCRIPTION
**Pull request by corywilson88**
# Relevant Story
[Research Safari Build#8](https://github.com/MSU-CS4360-WhiteHat/ecommerce-scam-detector/issues/8)

## Description 
Had to add `description` and `host_permissions` to `manifest.json` as they are required for safari extensions.

## Tests

Full list of [Restrictions](https://developer.apple.com/documentation/safariservices/safari_web_extensions/converting_a_web_extension_for_safari) we will need to be aware of. Most important at this stage of development: 

1. tabs
   - Extension must have host permission.
2. webNavigation
   - onCreatedNavigationTarget, onReferenceFragmentUpdated, onTabReplaced, onHistoryStateUpdated not supported. transitionType, transitionQualifiers not supported.

## Screenshots
Screen Shot 2023-02-07 at 5 48 59 PM
Screen Shot 2023-02-07 at 5 48 48 PM

### CODE REVIEW

**Review by LucasMali — APPROVED** [review]

File: manifest.json [hunk 1]
```diff
@@ -2,6 +2,7 @@
     "manifest_version": 2,
     "name": "Ecommerse Scam Detector",
     "version": "1.0",
+    "description": "An extension that tries to deted if a website is a scam or not",
```
Comment (thrandale) [inline manifest.json:5]: typo "deted"

File: manifest.json [hunk 2]
```diff
@@ -12,6 +13,10 @@
         "webNavigation",
         "tabs"
     ],
+    "host_permissions":[
```
Comment (thrandale) [inline manifest.json:16]: should probably also add "http" as wel

**Review by thrandale — CHANGES_REQUESTED** [review]
See inline comments

**Comment by corywilson88** [comment]
fixed both requests. good eyes!

**Pull request merged**